gentest/gentest-driver.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import * as fs from 'node:fs/promises';
+import {format} from 'node:util';
+import * as path from 'path';
+import * as process from 'node:process';
+import {Builder, logging} from 'selenium-webdriver';
+import {Options} from 'selenium-webdriver/chrome.js';
+
+await (async () => {
+  const options = new Options();
+  options.addArguments(
+    '--force-device-scale-factor=1',
+    '--window-position=0,0',
+    '--hide-scrollbars',
+  );
+  options.setLoggingPrefs({
+    browser: 'ALL',
+    performance: 'ALL',
+  });
+  const driver = await new Builder()
+    .forBrowser('chrome')
+    .setChromeOptions(options)
+    .build();
+
+  process.chdir(path.dirname(process.argv[1]));
+
+  const fixtures = await fs.readdir('./fixtures');
+  for (const fileName of fixtures) {
+    const fixture = await fs.readFile('fixtures/' + fileName, 'utf8');
+    const fileNameNoExtension = path.parse(fileName).name;
+    console.log('Generate', fileNameNoExtension);
+
+    // TODO: replace this with something more robust than just blindly replacing
+    // start/end in the entire fixture
```

Comment:
Probably the fixtures should be changed to use `inset-inline-start` and `inset-inline-end` instead of left/right? I don't think those existed when these tests were created.

Reply:
There are the other things like `margin-start` etc. I think we need a pretty wholistic system to replace things like that